src/components/Forms.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Categories from '../Pages/Categories';
+
+const FormInput = ({ addBookItem }) => {
+  const [book, setBook] = useState('');
+  const [category, setCategory] = useState('');
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (book.trim()) {
+      addBookItem(book, category); // Pass the book and category values
+      setBook('');
+      setCategory('');
+    }
+  };
+
+  const handleChange = (e) => {
+    setBook(e.target.value);
+  };
+
+  const handleCategoryChange = (selectedCategory) => {
+    setCategory(selectedCategory);
+  };
+
+  return (
+    <>
+      <form onSubmit={handleSubmit} className="form-container">
+        <input
+          type="text"
+          placeholder="Add a book"
+          value={book}
+          onChange={handleChange}
+        />
```

Comment:
- [ ] Well done :clap:  you did almost changes :+1:  but As I mentioned in the previous review :point_up_2: if you check the [requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v3_0.md) you will see there are mentioned `IMPORTANT in the design you can see an input for a category - please replace it with an author.` but in your case, you just add one input field so kindly add one more field for author and please replace the placeholder name `Add a book` with `title` but still you didn't implement these changes kindly fix these issues :slightly_smiling_face:

Reply:
Kindly elaborate @aamirkhan2478

Reply:
Please I can really get what you mean, pls can we have a zoom call?

Thanks